tests/specs/err/error-payload.e2e.js

```diff
@@ -53,8 +53,8 @@ describe('error payloads', () => {
 
     const { request: { body: { err } } } = await browser.testHandle.expectErrors()
 
-    expect(Math.abs(err[0].params.firstOccurrenceTimestamp - before)).toBeWithin(0, 20)
-    expect(Math.abs(err[0].params.firstOccurrenceTimestamp - after)).toBeWithin(0, 20)
+    expect(Math.abs(err[0].params.firstOccurrenceTimestamp - before)).toBeWithin(0, 50)
+    expect(Math.abs(err[0].params.firstOccurrenceTimestamp - after)).toBeWithin(0, 50)
```

Comment:
This test is no longer capturing the spirit of the original intent so we might need to re-evalute what it is doing.

Basically the test should just be checking that the firstOccurrenceTimestamp is the one that's used (which should be extremely close to the "before" stamp), and that the firstOccurenceTimestamp does not match the "after" which occurs later

Reply:
Care to take another look?